sdk/core/System.Net.ClientModel/tests/client/MapsClient/MapsClient.cs

```diff
@@ -6,52 +6,42 @@
 using System.Net.ClientModel;
 using System.Net.ClientModel.Core;
 using System.Text;
-using System.Threading;
 
 namespace Maps;
 
 public class MapsClient
 {
+    private const string LatestServiceVersion = "1.0";
+
     private readonly Uri _endpoint;
     private readonly KeyCredential _credential;
-    private readonly MessagePipeline _pipeline;
-    private readonly string _apiVersion;
+    private readonly PipelineOptions _pipelineOptions;
+    private readonly PipelinePolicy[] _clientPolicies;
```

Comment:
We need to keep _clientPolicies somewhere so when we create a new pipeline on RequestOptions we can pass them into it, since we won't copy them out of PipelineOptions' cached pipeline.

Reply:
This is a change the generator will have to handle